mercury_app/tests.py

```diff
@@ -136,6 +214,31 @@ def test_home(self):
         self.assertEqual(response.status_code, 302)
 
 
+class SelectEventsLoggedTest(TestBase):
+
+    def setUp(self):
+        super(SelectEventsLoggedTest, self).setUp()
+
+    @patch('mercury_app.views.get_api_organization', return_value=MOCK_ORGANIZATION_API.get('organizations'))
+    @patch('mercury_app.views.get_api_events_org', return_value=MOCK_EVENT_API.get('events'))
+    def test_screen_with_one_event(self, mock_get_api_events_org, mock_get_api_organization):
+        response = self.client.get('/select_events/')
+        self.assertEqual(response.status_code, 200)
+
+    @patch('mercury_app.views.get_api_events_id', return_value=MOCK_EVENT_API.get('events')[0])
```

Comment:
Please do not mock just one event, when the endpoint will return a list of events.